### PR TITLE
improve user experience by adding a message when geosafe page is loaded

### LINKED ISSUE #161: improve analysis loading experience
While it loads, the analysis pages remains blank for a long pause and shows the legend skeleton.

Manage loading so something shows immediately or at least have a spinner or message. 

Don't show the legend until it's populated.

### PR DESCRIPTION
**Pull request by boney-bun**
fix #161
here is the demo:
enhanceuserexp161

### CODE REVIEW

File: templates/geosafe/analysis/legend.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="legend" onmouseover="map.dragging.disable()" onmouseout="map.dragging.enable();">
```
Comment (lucernae) [inline templates/geosafe/analysis/legend.html:1]: Nice! It's good having it refactored and separated.

**Review by lucernae — APPROVED** [review]
LGTM, @boney-bun!
Can't see apparent issue with this.

File: templates/geosafe/analysis/create.html [hunk 2]
```diff
@@ -492,14 +474,33 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
...
+            $("#legend-hazard-body").toggle(is_hazard);
+            $("#legend-impact-body").toggle(is_impact);
+            //hide/view legend
+            $("#legend-title").click(function () {
+                $("#legend-body").toggle();
+            });
+
+            // Add scrollbar in case of overflowing in dimension-y
+            $("#legend-body").css("max-height", 0.7 * w_height);
+            $("#legend-body").css("overflow-y", "auto")
```
Comment (lucernae) [inline templates/geosafe/analysis/create.html:502]: If this is always the case, why not put this on css:

```
#legend-body{
   overflow-y: auto
}
```

Comment (boney-bun) [reply templates/geosafe/analysis/create.html:502]: i think this one is from another issue.
i'll separate them when doing another ticket.

**Pull request merged**